### DISCUSSION TITLE
video stream and/or snapshots

### DISCUSSION THREAD

**Discussion by mikkokunnari** [SudoMagicCode/SudoSignals#11] [v1 - RFE & Feature Requests]
Would be nice to have some snapshots/screencapture from installation computer.  
Also have this in sudosignals component, so you could add few tops and see them in sudosignals dashboard. 
even really low fps could help to see what is running on installation PC.

**Comment by raganmd** [comment]
Love this request - we've been talking about what this might look like for a bit.
